bot.py

```diff
@@ -467,10 +468,28 @@ async def show_queue(self, message, _command_content):
 
         await message.channel.send(reply)
 
+    async def move(self, message, _command_content):
+        """
+        Moves the bot to the voice channel that the message author is currently
+        connected to.
+        """
+        if not await self.create_or_get_voice_client(message):
+            return
```

Comment:
Something which might not matter: If I'm in a voice channel, bot is in no voice channel, and I call `!move`, the bot will join my voice channel with nothing to play.

Reply:
Good point. Probably doesn't matter but it's not expected behaviour. 
What do you think should happen? Just a simple error message maybe?

Reply:
Haha I think the second part is unnecessary, but I'll add a message.